infra/src/main/kotlin/fi/fta/geoviite/infra/linking/switches/SwitchLinkingService.kt

```diff
@@ -1181,18 +1188,41 @@ fun getSwitchAlignmentJointSequences(switchStructure: SwitchStructure): List<Lis
     return switchStructure.alignments.map { alignment -> alignment.jointNumbers }
 }
 
-fun getAllSwitchAlignmentJointSequences(switchStructure: SwitchStructure): List<List<JointNumber>> {
-    return getSwitchAlignmentJointSequences(switchStructure) + getPartialSwitchAlignmentJointSequences(switchStructure)
-}
+fun validateJointSequence(
+    switchStructure: SwitchStructure,
+    jointsOnLocationTrack: List<JointOnEdge>,
+    geometry: LocationTrackGeometry,
+): List<JointOnEdge> {
```

Comment:
Wanhempaa perua, mutta jäin miettimään tämän nimeämistä 🤔 Tuntuu hieman oudolta, että `validate`-alkuinen funktio palauttaa listan jointteja. Koska paluuarvona näyttäisi olevan sisääntuleva parametri tai tyhjä lista, niin ihan joku true/false kuulostaisi fiksulta paluuarvolta (kutsujallahan on jo lista jointeista, koska se antaa ne sisään.)